yarn.lock

```diff
@@ -2068,6 +2068,11 @@ anymatch@^3.0.3:
     normalize-path "^3.0.0"
     picomatch "^2.0.4"
 
+app-root-path@^2.2.1:
```

Comment:
AFAICT, we've not added any new external dependencies. This file shouldn't be changing. Can you check?

Reply:
Solved by pulling from master